docs/reference/errors-and-warnings/NU1212.md

```diff
@@ -0,0 +1,40 @@
+---
+title: NuGet Error NU1212
+description: NU1212 Error code
+author: nkolev92
+ms.date: 02/02/2022
+ms.topic: reference
+f1_keywords: 
+  - NU1212
+---
+
+# NuGet Warning NU1212
+
+## Scenario 1
+
+<pre>Invalid project-package combination for Contoso.Library 1.0.0. DotnetToolReference project style can only contain references of the DotnetTool type</pre>
+
+### Issue
+
+DotnetToolReference project style can only contain references of the [DotnetTool](../../create-packages/set-package-type.md#known-package-types) type, and either the project style is not `DotnetToolReference` or the package is not of `DotnetTool` type.
```

Comment:
```suggestion
`DotnetToolReference` project style can only reference .NET tools with the [DotnetTool](../../create-packages/set-package-type.md#known-package-types) package type. Either the project style is not `DotnetToolReference` or a referenced package is not of `DotnetTool` type.
```

Reply:
Love it. Merged. I'll copy it in the scenario 2 as well.